astropy/io/votable/table.py

```diff
@@ -221,13 +222,15 @@ def validate(source, output=None, xmllint=False, filename=None):
 
     Parameters
     ----------
-    source : str or readable file-like object
+    source : path-like or file-like
```

Comment:
Elsewhere you left it as `str`, not `path-like`. Maybe here too, just for consistency?

Reply:
I did that here because the description says it works on `pathlib.Path`, which is path-like, not str.

Reply:
A good followup would be to make all (or as much as possible) accept path-like

Reply:
The problem is that most of the descriptions say that - hence I thought one might as well be consistent with `str` for now...

Reply:
Good catch. I think I found the rest of the ones in the doc.

Reply:
They're all path-like 👍